### PR TITLE
update dnn cfd project description

### PR DESCRIPTION
**Pull request by MarioRuettgers**
Maybe we need to update the project members?

Rakesh, Rishabh, Onishi and Hadrien are new.

### CODE REVIEW

**Comment by pancetta** [comment]
Sorry for being late here. There is still a bug in the `persons.yaml` or a typo in the partner list. Please fix.

File: _projects/dnn_cfd.md [hunk 1]
```diff
@@ -13,47 +13,29 @@ topics:
...
-  - generative adversial networks
-  - transfer learning
+  - physics-informed neural networks
+  - reduced order models
+  - graph neural networks
 head: lintermann_a
 members:
   - tsubokura_m
-  - onishi_k
+  - onishi_y
```
Comment (pancetta) [inline _projects/dnn_cfd.md]: Are you sure about the change in the first name here? If so, you need to edit this in the database, too.

Comment (MarioRuettgers) [reply _projects/dnn_cfd.md]: thanks, I have updated the database accordingly

Comment (MarioRuettgers) [reply _projects/dnn_cfd.md]: Should I do the pull request again?

File: _data/people.yml [hunk 2]
```diff
@@ -2182,6 +2209,15 @@ prabhu_t:
   email: tprabhu2@illinois.edu
   homepage: http://web.engr.illinois.edu/~tprabhu2/
 
+puri_r:
+  sur_name: Puri
+  given_name: Rishabh
+  affiliation: Engler-Bunte Institute - Combustion Technology, Karlsruhe Institute for Technology (KIT)
```
Comment (pancetta) [inline _data/people.yml]: cannot have that name. Use `external` here

Comment (MarioRuettgers) [reply _data/people.yml]: OK, I have modified that

**Pull request merged**